### DISCUSSION TITLE
website: Add copy button on the config titles.

### DISCUSSION THREAD

**Discussion by lesleyfon** [ghostty-org/ghostty#4230] [Feature Requests, Ideas] [answered]
I thought it would be great to have a `copy button` next to the title on the config reference page. This would make it easy to click and copy a config instead of highlighting it before copying.

<img width="804" alt="Screenshot 2024-12-31 at 2 46 19 PM" src="https://github.com/user-attachments/assets/94ebe7b4-0bae-428d-b22d-89c1d8390b3e" />

@mitchellh thoughts?

**Comment by mitchellh** [comment, marked as answer]
I like this. It'd be good to make this configurable based on the front matter or something so that this only applies to the reference pages. I'm not sure how else to do it but some way to make sure it's not _every header_ would be good. I'm supportive.

**Comment by lesleyfon** [comment]
> I like this. It'd be good to make this configurable based on the front matter or something so that this only applies to the reference pages. I'm not sure how else to do it but some way to make sure it's not _every header_ would be good. I'm supportive.

I have a local implementation that applies only to the reference pages and not to every header on the website.
